pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -1282,7 +1414,12 @@ func (testCases ListUsersTests) runListUsersTestCases(t *testing.T) {
 			test.req.StoreId = storeID
 
 			resp, err := l.ListUsers(ctx, test.req)
-			require.ErrorIs(t, err, test.expectedError)
+
+			actualErrorMsg := ""
+			if err != nil {
+				actualErrorMsg = err.Error()
+			}
+			require.Equal(t, test.expectedErrorMsg, actualErrorMsg)
```

Comment:
I had issue wrapping the error here. Really we just want to assert a specific message, not that literal error itself is equal.